CHANGELOG.md

```diff
@@ -2,335 +2,334 @@
+- This will force the current administrators to change their passwords after 90 days has passed from the previous login.
+- For development/testing/staging environments this also means that the default user passwords have changed to `decidim123456789` to match the minimum length rules for admins.
+- For consistency reasons, regular users password has also been changed with the seed data.
 
-Documentation is also updated so be sure to check the options in the [Environment Variables](https://docs.decidim.org/en/configure/environment_variables/) doc.
+The relevant [Environment Variables](https://docs.decidim.org/en/configure/environment_variables/) are:
 
-Note that this change does not affect existing installations as only the `config/initializers/decidim.rb` and `config/secrets.yml` files are involved.
-However you can migrate to the new structure easily by creating a new Decidim app and copying or adapting those files to your own project.
+| Name | Value | Default value |
+| -------- | -------- | -------- |
+| DECIDIM_ADMIN_PASSWORD_STRONG     | Enable strong password rules for admin users.     | true     |
+| DECIDIM_ADMIN_PASSWORD_EXPIRATION_DAYS | Defines how many days admin passwords are valid before they need to be reset. | 90 |
+| DECIDIM_ADMIN_PASSWORD_REPETITION_TIMES | Defines how many previous passwords are compared against new admin user passwords. | 5 |
+| DECIDIM_ADMIN_PASSWORD_MIN_LENGTH | The minimum character length for admin user passwords. | 15 |
 
-#### Reminders for pending orders in budgets
+You can read more about this change on PR [\#9347](https://github.com/decidim/decidim/pull/9347).
 
-**decidim-core**, **decidim-budgets**: Reminders for pending orders in budgets [#8621](https://github.com/decidim/decidim/pull/8621). To generate reminders:
+#### 2.8 Service workers
 
-```bash
-bundle exec rake decidim:reminders:all
+For the Progressive Web Application related features, like Push Notifications and Add To Home Screen, you'll need to update your webpack configuration:
+
+```console
+bin/rails decidim:webpacker:install
 ```
 
-Or add cronjob:
+You'll need to also add these to your .gitignore:
 
-```bash
-4 0 * * * cd /home/user/decidim_application && RAILS_ENV=production bundle exec rake decidim:reminders:all
+```gitignore
+public/sw.js
+public/sw.js.map
 ```
 
-#### New Comments statistics structure
+These files will be generated by the asset compilation task in your production server. Most of the time this should be handled automatically by your deployment process (like Capistrano or Heroku). In case that you need to run that manually, this is the command:
 
-PR [#8012](https://github.com/decidim/decidim/pull/8012) Participatory space to comments, to fix the statistics. Use
-`rake decidim_comments:update_participatory_process_in_comments` to migrate existing comments to the new structure.
+```console
+bin/rails assets:precompile
+```
 
-#### New Api Documentation engine
+### 3. One time actions
```

Comment:
You mentioned in the last weekly that you were not perfectly satisfied with this heading. I could not come up with any better either but here are some alternative suggestions that you can consider:
- Manual chores (or just Chores)
- Call to action (or CTA or Actions)
- Configuration
- Refinements (can be ambiguous)
- Finalization

Feel free to dismiss if the current one is better. Just tried to come up with some alternatives.

Reply:
I prefer the ones we have. Thanks for the options, though, I'll take this into account in the next release.